flows/interfaces.go

```diff
@@ -218,6 +218,10 @@ type LogEntry interface {
 	Event() Event
 }
 
+type FlowStack interface {
+	HasFlow(FlowUUID) bool
```

Comment:
This is used by StartFlowAction... otherwise flowStack could be completely internal to the engine package